src/modules/sway/workspaces.cpp

```diff
@@ -1,9 +1,28 @@
 #include "modules/sway/workspaces.hpp"
 
+#include <string>
+#include <cctype>
 #include <spdlog/spdlog.h>
 
+
 namespace waybar::modules::sway {
 
+// this is the code that sway uses to assign a number to a workspace. This is
+// taken quite verbatim from `sway/ipc-json.c`.
+int sway_wsname_to_num(std::string name) {
```

Comment:
Can you just move this func as a method of Workspaces class, and i'll merge this PR :)

Reply:
Is it really needed? The the lambda used in the `std::sort` call would then need to capture `this` as well.

Reply:
At least a static method, that's just in order to avoid anonymous func outside the class.

Reply:
@Alexays done :)